tutorials/overview.md

```diff
@@ -1,41 +0,0 @@
----
-description: This page provides a comprehensive overview of tutorials available for Rollkit.
----
-
-# Tutorials
-
-Welcome to the Rollkit tutorials section! Here, you'll find a wide range of
-tutorials and guides designed to help you understand and effectively use
-Rollkit, the first sovereign rollup framework.
-
-Whether you're a beginner just starting out, an intermediate user looking
-to expand your knowledge, or an advanced user seeking to delve into more
-complex topics, we've got you covered. Our tutorials are categorized based
-on the level of expertise required, so you can easily find the resources
-that best suit your needs.
-
-In this section, you'll find:
-
-* Rollkit
-  * [GM world rollup: Part 1, local devnet](/tutorials/gm-world.md)
-  * [GM world frontend](/tutorials/gm-world-frontend.md)
-  * [Wordle app](/tutorials/wordle.md)
-* DA
-  * [GM world rollup: Part 2, mocha testnet](/tutorials/gm-world-mocha-testnet.md)
-  * [GM world rollup: Part 3, mainnet](/tutorials/gm-world-mainnet.md)
-* Execution
-  * [CosmWasm rollup](/tutorials/cosmwasm.md)
-  * [Polaris EVM rollup](/tutorials/polaris-evm.md)
-
-## 💻 Support {#support}
-
-The tutorials will explore developing with Rollkit,
-which is still in Alpha stage. If you run into bugs, please write a Github
```

Comment:
with removing this, we should make sure that this call out of Alpha is on all the tutorials. 
@jcstein is there a way to refactor a callout/tip so that we could define this alpha warning once and import it to all the tutorial pages?

Reply:
we could make a component but i honestly think it is much easier to [use the callouts that are provided in vitepress](https://vitepress.dev/guide/markdown#github-flavored-alerts)

Reply:
@yarikbratashchuk can you pleas make sure that this is sufficiently called out in the tutorials now?

Reply:
making an issue to follow up on this, so we can get this merged https://github.com/rollkit/docs/issues/362

Reply:
https://github.com/rollkit/docs/pull/360/commits/ec5e26488a597ee8d3064746c6fdef7e2f5c27b1

Reply:
made it a component in https://github.com/rollkit/docs/pull/360/commits/4c8251e83a2bc1101bb4edae5c1b595cb53619b8